IdentityCore/src/controllers/ios/MSIDBrokerInteractiveController.m

```diff
@@ -187,9 +188,18 @@ - (void)saveToPasteBoard:(NSURL *)url
     [appPasteBoard setURL:url];
 }
 
-+ (BOOL)completeAcquireToken:(NSURL *)resultURL
-       brokerResponseHandler:(MSIDBrokerResponseHandler *)responseHandler
++ (BOOL)completeAcquireToken:(nullable NSURL *)resultURL
```

Comment:
Q: So, is this something developers will eventually need to call (through MSAL/ADAL) in app delegate?

Reply:
yeah that is what i meant :p
something the they will call msal that will call this :)